packages/celotool/src/lib/vm-testnet-utils.ts

```diff
@@ -339,15 +266,22 @@ export async function generateAndUploadSecrets(celoEnv: string) {
   // Tx Nodes
   const txNodeCount = parseInt(fetchEnv(envVar.TX_NODES), 10)
   for (let i = 0; i < txNodeCount; i++) {
-    const secrets = generateValidatorSecretEnvVars(AccountType.TX_NODE, i)
+    const secrets = generateNodeSecretEnvVars(AccountType.TX_NODE, i)
     await uploadSecrets(celoEnv, secrets, `tx-node-${i}`)
   }
   // Validators
   const validatorCount = parseInt(fetchEnv(envVar.VALIDATORS), 10)
   for (let i = 0; i < validatorCount; i++) {
-    const secrets = generateValidatorSecretEnvVars(AccountType.VALIDATOR, i)
+    const secrets = generateNodeSecretEnvVars(AccountType.VALIDATOR, i)
     await uploadSecrets(celoEnv, secrets, `validator-${i}`)
   }
+  // Proxies
```

Comment:
What do you think of adding an option to specify the percentage of validators that are proxied?  So that would allow for a testnet with some proxied validator and some standalone ones.

Reply:
The way it's set up you can set how many of the validators you want to be proxied, is that ok for what you were thinking? I imagine introducing percentages might slightly obfuscate what's going on

So for example
```
VALIDATORS=4
PROXIED_VALIDATORS=3
```
Would have 75% of the validators be proxied